compiler/modules/types/bdd.bal

```diff
@@ -227,3 +227,15 @@ isolated function bddToString(Bdd b, boolean inner = false) returns string {
         return str;
     }
 }
+
+// This is for debug purposes. Used by regex to add comments for type definitions
+public isolated function semTypeToBddString(SemType ty) returns string? {
+    if ty !is ComplexSemType {
+        return ();
+    } 
+    var currentSubtype = ty.subtypeDataList[0];
```

Comment:
This isn't in general going to be right.

I think it's good to have public function in core that gives a human understandable version of the semType, but it needs to be a bit more sophisticated than this, if we are going to merge it in.